evalml/automl/automl_search.py

```diff
@@ -609,6 +616,8 @@ def _compute_cv_scores(self, pipeline, X, y):
         cv_data = []
         logger.info("\tStarting cross validation")
         for i, (train, test) in enumerate(self.data_split.split(X, y)):
+            if pipeline.model_family == ModelFamily.ENSEMBLE and i > 0:
```

Comment:
Don't run all folds for ensemble methods, only run first to use the same data split.

Reply:
Is this for performance reasons? Would we get rid of this if we implemented our own stacked ensembler? Might be helpful to add a comment here.

Reply:
Oooh. This is slick.

Reply:
Can we add a log message in this case? Just so its extra clear that we don't support CV for ensemble methods, and will override default or user-specified behavior

Making a mental note that we should add this to the user guide for stacking

Reply:
Yup @freddyaboulton it's for performance reasons--we don't do CV on the ensemble (since it internally does CV). Will add a comment + log a message!